source/indexes/create-search-index.txt

```diff
@@ -0,0 +1,67 @@
+.. _compass-create-search-index:
+
+============================
+Create an Atlas Search Index
+============================
+
+You can create Atlas Search indexes in |compass|. Atlas Search indexes
+let you query data in `Atlas Search
+<https://www.mongodb.com/docs/atlas/atlas-search/>`__. Atlas Search
+indexes enable performant text search queries by mapping search terms to
+the documents that contain those terms.
+
+About this Task
+---------------
+
+To create an Atlas Search index, your deployment must be hosted on
+`MongoDB Atlas <https://www.mongodb.com/docs/atlas/>`__ and have a
+cluster tier of M10 or higher.
+
+Steps
+-----
+
+.. procedure::
+   :style: normal
+   
+   .. step:: Open the index creation dialog
+
+      From the :ref:`Indexes <collection-tab>` tab, click the
+      :guilabel:`Create` button, then click :guilabel:`Search Index`.
+
+   .. step:: Enter the index name
+
+      Specify a name for the index.
+
+   .. step:: Specify the search index definition
+
+      |compass-short| provides templates for different kinds of search
+      indexes. To learn more about search index definition syntax, see
+      :ref:`search-index-definition-create-mongosh`.
+
+   .. step:: Click :guilabel:`Create Search Index`
+
```

Comment:
Should we add some additional context for monitoring the Index build after clicking create? For instance steps 11 and 12 on the Atlas documentation provide some more details after you click create about how to check in on the index creation process.

https://www.mongodb.com/docs/atlas/atlas-search/create-index/#close-the-you-re-all-set-modal-window

Reply:
I will ask during tech review if there is any kind of notification we'll want to mention here.